src/test/java/com/xceptance/neodymium/tests/BrowserStatementTest.java

```diff
@@ -125,6 +202,17 @@ public void testClassBrowserSuppressed() throws Throwable
         checkDescription(ClassBrowserSuppressed.class, expected);
     }
 
+    @Test
+    public void testDisableRandomBrowserAnnotation() throws Throwable
+    {
+        //
+        String[] expected = new String[]
+        {
+          "test1"
+        };
+        checkDescription(DisableRandomBrowserAnnotation.class, expected);
```

Comment:
it would be good to have a test that actually disables something ;-)
please add more test functions inside this test e.g.

```
    @RandomBrowsers(0)
    @Browser("Chrome_1024x768")
    @Browser("Chrome_1500x1000")
    @Test
    public void test2()
    {

    }
```

Reply:
test extended